migration-engine/core/src/cli.rs

```diff
@@ -119,6 +113,37 @@ fn create_conn(
     }
 }
 
+/// Try to connect as an admin to a postgres database. We try to pick a default database from which
+/// we can create another database.
+fn create_postgres_admin_conn(mut url: Url) -> crate::Result<SqlMigrationConnector> {
+    let candidate_default_databases = &["postgres", "template1"];
```

Comment:
Can't we just connect to template1? Or are there cases where it's missing?

Reply:
I kept `postgres` because I'm not sure about the permissions `template1` is not really meant to be used directly so sometimes it could be more locked down than `postgres`. The DigitalOcean managed postgres for example has a locked down `template1`. On my machine I can log onto it, so I thought it's safer to try both. What do you think?

Reply:
By "locked down" I mean "can't be logged into with the user's account"

Reply:
Just makes the connection slower. Although only if postgres db doesn't exist.